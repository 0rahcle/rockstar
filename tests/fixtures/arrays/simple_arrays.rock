Rock B
Shout B

Rock X with 1, 2, 3
Shout X

Let Y be 2
Shout Y
Rock Y
Shout Y
Rock Y with 3, 4, 5
Shout Y

Tommy says hey
Gina says boy 
Black Betty says wham ba lam 
Rock Tommy with Gina, Black Betty
Shout Tommy
