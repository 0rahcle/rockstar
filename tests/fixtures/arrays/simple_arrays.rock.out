[]
[ 1, 2, 3 ]
2
[ 2 ]
[ 2, 3, 4, 5 ]
[ 'hey', 'boy ', 'wham ba lam ' ]
